contracts/tests/MACI.test.ts

```diff
@@ -330,9 +237,25 @@ describe("MACI", () => {
       expect(receipt?.status).to.eq(1);
     });
 
-    it("the state root must be correct", async () => {
+    it("should have the correct state root on chain after merging the acc queue", async () => {
       const onChainStateRoot = await stateAqContract.getMainRoot(STATE_TREE_DEPTH);
       expect(onChainStateRoot.toString()).to.eq(maciState.stateTree.root.toString());
     });
   });
+
+  describe("getStateAqRoot", () => {
+    it("should return the correct state root", async () => {
+      const onChainStateRoot = await maciContract.getStateAqRoot();
+      expect(onChainStateRoot.toString()).to.eq(maciState.stateTree.root.toString());
+    });
+  });
+
+  describe("getPoll", () => {
+    it("should return an address for a valid id", async () => {
+      expect(await maciContract.getPoll(pollId)).to.not.eq(0);
+    });
+    it("should throw when given an invalid poll id", async () => {
+      await expect(maciContract.getPoll(5)).to.be.revertedWithCustomError(maciContract, "PollDoesNotExist").withArgs(5);
+    });
```

Comment:
```suggestion
    });

    it("should throw when given an invalid poll id", async () => {
      await expect(maciContract.getPoll(5)).to.be.revertedWithCustomError(maciContract, "PollDoesNotExist").withArgs(5);
    });
```

Reply:
prettier just puts it back to how it was 😆